yarn.lock

```diff
@@ -4528,22 +4528,22 @@ __metadata:
   linkType: hard
 
 "typescript@npm:^4.8.4":
-  version: 4.8.4
-  resolution: "typescript@npm:4.8.4"
+  version: 4.9.5
```

Comment:
Do we need to bump TypeScript for these changes to work?

Reply:
The version in `package.json` has not been changed, but as we pin the major (^), the latest 4.x gets pulled - I don't think we need to update Typescript in this PR necessarily though

Reply:
Yeah, maybe we ought to not bump TypeScript here, and then pin TypeScript to `~4.8.4` in another PR? This would match what we tend to do for other libraries.

Reply:
Looks like expressions similar to `'key' in encryptionKey` don't work like expected with 4.8.x
I'm still trying to figure that out but Typescript doesn't infer that the object is of a type that has that property.

To work around with that, I added `@metamask/utils` as dependency and used `isPlainObject` and `hasProperty` in the typeguards

Reply:
I also opened another PR to pin the minor version: https://github.com/MetaMask/browser-passworder/pull/50